test/UnitTests/Microsoft.Testing.Platform.UnitTests/Program.cs

```diff
@@ -66,3 +66,41 @@ public async Task AfterRunAsync(int returnValue, CancellationToken cancellationT
 
     public Task BeforeRunAsync(CancellationToken cancellationToken) => Task.CompletedTask;
 }
+
+internal sealed class FakeTrxReportGeneratorCommandLine : ICommandLineOptionsProvider
```

Comment:
Was it only for debugging purposes?

Reply:
The runner in parent repo provides those parameters, but we cannot use the reporter because there are breaking changes in the platform. If the provider is missing the run exits with "unknown parameter". So I removed the real provider and added this fake one for now. 

I described the exact issue in our internal chat.